src/components/inputs/react-hook-form/ag-grid-table/custom-ag-grid-table.tsx

```diff
@@ -227,6 +236,24 @@ export const CustomAgGridTable: FunctionComponent<CustomAgGridTableProps> = ({
         }
     };
 
+    const onCellEditingStopped = useCallback(
+        (event: CellEditingStoppedEvent) => {
+            const rowIndex = getIndex(event.data);
+            if (rowIndex === -1) {
+                return;
+            }
+            update(rowIndex, event.data);
+        },
+        [getIndex, update]
+    );
+
+    const onSortChanged = useCallback((event: SortChangedEvent) => {
+        const isAnycolumnhasSort = event?.api
+            ?.getColumnState()
```

Comment:
I think the optional chaining isn't necessary here since event can't be undefined

Reply:
Done